tests/conftest.py

```diff
@@ -64,7 +64,7 @@
 from invenio_vocabularies.records.api import Vocabulary
 
 from invenio_rdm_records import config
-from invenio_rdm_records.proxies import current_rdm_records
+from invenio_rdm_records.proxies import current_rdm_records, current_rdm_records_service
```

Comment:
There is only one appearance of `current_rdm_records` and it is to access the `.service`, the new import can be used.
```suggestion
from invenio_rdm_records.proxies import current_rdm_records_service
```